docs_user/modules/con_adoption-prerequisites.adoc

```diff
@@ -19,18 +19,18 @@ Planning information::
 Back-up information::
```

Comment:
Why is there two sets of colons here, :: ?

Reply:
So that it renders as a bold heading in the published docs.